f5lbaasdriver/v2/bigip/service_builder.py

```diff
@@ -73,12 +73,23 @@ def build(self, context, loadbalancer, agent, **kwargs):
                 context,
                 loadbalancer
             )
-
-            service['qos'] = self._get_extended_qos(
+ 
```

Comment:
should delete this empty space at line 76

Reply:
resolved